### ISSUE TITLE
Local scene file support on Android

### ISSUE THREAD

**Issue by Buthrakaur** [tangrams/tangram-es#1564]
Loading of scene files works from assets and web, but doesn't seem to work for ordinary local filesystem files even though local files as sources are supported for offline map sources. It sounds like a trivial task - could the feature be added, please? Or am I doing something wrong?

Here is what I was trying. I ensured I have read/write external storage permissions and that the file really exists on the device:
`map.loadSceneFile("file:///storage/emulated/0/Download/scene.zip", sceneUpdates);`
=>
```
07-10 14:46:23.281 24878-24878/com.mapzen.tangram.android D/tangram: new File("/storage/emulated/0/Download/scene.zip").exists(): true
07-10 14:46:23.281 24878-24878/com.mapzen.tangram.android D/tangram: new File("/storage/emulated/0/Download/scene.zip").canRead(): true
07-10 14:46:23.390 24878-24894/com.mapzen.tangram.android D/Tangram: TANGRAM tangram.cpp:187: Loading scene file: file:///storage/emulated/0/Download/scene.zip
07-10 14:46:23.391 24878-24894/com.mapzen.tangram.android D/Tangram: WARNING platform.cpp:29: Failed to read file at path: file:///storage/emulated/0/Download/scene.zip
```

`map.loadSceneFile("file://storage/emulated/0/Download/scene.zip", sceneUpdates);`
=>
```
07-10 14:48:31.657 24878-24878/com.mapzen.tangram.android D/tangram: new File("storage/emulated/0/download/scene.zip").exists(): true
07-10 14:48:31.657 24878-24878/com.mapzen.tangram.android D/tangram: new File("storage/emulated/0/download/scene.zip").canRead(): true
07-10 14:48:31.657 24878-24878/com.mapzen.tangram.android D/Tangram: TANGRAM tangram.cpp:187: Loading scene file: file://storage/emulated/0/download/scene.zip
07-10 14:48:31.657 24878-24878/com.mapzen.tangram.android D/Tangram: WARNING platform.cpp:29: Failed to read file at path: file://storage/emulated/0/download/scene.zip
```

`map.loadSceneFile("storage/emulated/0/Download/scene.zip", sceneUpdates);`
=>
```
07-10 14:49:01.327 24878-24878/com.mapzen.tangram.android D/tangram: new File("storage/emulated/0/download/scene.zip").exists(): true
07-10 14:49:01.327 24878-24878/com.mapzen.tangram.android D/tangram: new File("storage/emulated/0/download/scene.zip").canRead(): true
07-10 14:49:01.327 24878-24878/com.mapzen.tangram.android D/Tangram: TANGRAM tangram.cpp:187: Loading scene file: asset:///storage/emulated/0/download/scene.zip
07-10 14:49:01.330 24878-24878/com.mapzen.tangram.android D/Tangram: WARNING sceneLoader.cpp:220: No source defined in the yaml scene configuration.
```
(the component thinks the path is asset path)

`map.loadSceneFile("/storage/emulated/0/Download/scene.zip", sceneUpdates);`
=>
```
07-10 14:50:24.610 24878-24878/com.mapzen.tangram.android D/tangram: new File("/storage/emulated/0/download/scene.zip").exists(): true
07-10 14:50:24.610 24878-24878/com.mapzen.tangram.android D/tangram: new File("/storage/emulated/0/download/scene.zip").canRead(): true
07-10 14:50:24.610 24878-24878/com.mapzen.tangram.android D/Tangram: TANGRAM tangram.cpp:187: Loading scene file: asset:///storage/emulated/0/download/scene.zip
07-10 14:50:24.611 24878-24878/com.mapzen.tangram.android D/Tangram: WARNING sceneLoader.cpp:220: No source defined in the yaml scene configuration.
```
(the component thinks the path is asset path)

**Comment by matteblair** [comment]
This is fixed as part of https://github.com/tangrams/tangram-es/pull/1465, so I would expect it to go into a new release very soon. Thanks for bringing this back to our attention!

**Comment by matteblair** [comment]
Resolved by #1659
